src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -6952,6 +6952,36 @@ void test_wdb_global_assign_agent_group_success(void **state) {
     __real_cJSON_Delete(find_group_resp);
 }
 
+void test_wdb_global_assign_agent_group_invalid_group_name(void **state) {
+    #define GROUPS_SIZE 10
```

Comment:
I'd wouldn't have a pre-processor definition inside a method. But this seems to be older than this PR.

Reply:
I use this "define" from other tests, and I think the main purpose of this definition is to give clarity to the test case by telling what that number represents. It simply will replace the string GROUPS_SIZE by 10 during the pre-processor but the code is more readable.

Reply:
I changed as we discussed.